app/processing/funding/get-fundings-by-calculation-id.js

```diff
@@ -0,0 +1,22 @@
+const db = require('../../data')
+
+const getFundingsByCalculationId = async (calculationId) => {
+  return db.funding.findAll({
+    attributes: [
+      'calculationId',
+      'areaClaimed',
+      'rate'
+    ],
+    include: [{
+      model: db.fundingOption,
+      as: 'fundingOptions',
+      attributes: ['name']
+    }],
+    where: {
+      calculationId
+    },
+    raw: true

```

Comment:
You could also use `nest: true` here which would mean that `fundingOptions` `name` attribute would be available as `fundingOptions.name` rather than `'fundingOptions.name'`.

Makes the syntax a bit neater downstream in my opinion but leave it up to your judgement.

Reply:
Recommendation applied, many thanks